src/modules/layerManager/i18n/layerManager.provider.js

```diff
@@ -11,7 +11,12 @@ export const provide = (lang) => {
 				layerManager_expand: 'expand properties',
 				layerManager_move_up: 'move Layer up',
 				layerManager_move_down: 'move Layer down',
-				layerManager_remove: 'remove Layer'
+				layerManager_remove: 'remove Layer',
+				layerManager_duplicate: 'duplicate Layer',
+				layerManager_layer_copy: 'Copy',
```

Comment:
clone

Reply:
the key `layerManager_layer_copy` is used as suffix for the label of a cloned layer, i guess '[Layername] Copy' is more familiar to the user than '[Layername] Clone'